<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentInfo</class>
 <widget class="QDialog" name="StudentInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Student Information</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#StudentInfo {
        background-color: #fef6f0   ;
        border: 1px solid black;
        border-radius: 10%;
}</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>431</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#titleLabel {
        font-size: 25px;
color: #2c3e50 ;
}</string>
   </property>
   <property name="text">
    <string>Student Profile</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>411</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#btnClose {
    background-color: #ff6b6b;
    border: 1.5px solid black;
    border-radius: 10px;
    padding: 6px;
    font-size: 16px;
}

/* Hover effect */
QPushButton#btnClose:hover {
    background-color: #ee5253;
}

</string>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditClass">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit#lineEditClass {
    color: black;;
    border: 1.5px solid black;;
    border-radius: 10px;
        font-size: 16px;
        padding: 0px 8px 0px 8px;
}
</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditName">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit#lineEditName {
    color: black;;
    border: 1.5px solid black;;
    border-radius: 10px;
        font-size: 16px;
        padding: 0px 8px 0px 8px;
}
</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_id">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#label_id {
        font-size: 16px;
        color: #2c3e50 ;
}</string>
   </property>
   <property name="text">
    <string>Student ID:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditId">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit#lineEditId {
    color: black;;
    border: 1.5px solid black;;
    border-radius: 10px;
        font-size: 16px;
        padding: 0px 8px 0px 8px;
}
</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_class">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#label_class {
        font-size: 16px;
        color: #2c3e50 ;
}</string>
   </property>
   <property name="text">
    <string>Class</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#label_name {
        font-size: 16px;
        color: #2c3e50 ;
}</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_password">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#label_password {
        font-size: 16px;
        color: #2c3e50 ;
}</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPassword">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit#lineEditPassword {
    color: black;;
    border: 1.5px solid black;;
    border-radius: 10px;
        font-size: 16px;
        padding: 0px 8px 0px 8px;
}
</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Normal</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
